linode/manager
# M3-5519: Add PayPal as recurring payment method

## Description
![Screen Shot 2021-10-26 at 12 05 04 PM](https://user-images.githubusercontent.com/6440455/138917771-f16baf3e-e0c0-4dd1-bf31-c5881707cec4.png)

- Adds ability to add PayPal as a recurring payment method
- Uses `react-paypal-js` library

## How to test

- Point Cloud Manager to dev or local dev
- Go to `/account/billing/add-payment-method`
- Try adding a PayPal account
  - This opens `sandbox.paypal.com` so you need to use a sandbox customer account (check your password manager)
